Modules/Invoice/Notifications/GoogleChat/SendPaymentReceivedNotification.php

```diff
@@ -28,9 +28,9 @@ public function via($notifiable)
         ];
     }
 
-    public function toGoogleChat($invoiceNotificationData)
+    public function toGoogleChat($notifiable)
     {
-        return GoogleChatMessage::create()
-            ->mentionAll('', " We have received the payment for{{$invoiceNotificationData}} successfully!\n");
+        return GoogleChatMessage::create()->mentionAll('', 'We have received the payment for' .
```

Comment:
Add extra space so that there should be proper spacing between project name and the words before and after it
```suggestion
        return GoogleChatMessage::create()->mentionAll('', 'We have received the payment for ' .
```

Reply:
Done